packages/blade/src/components/Typography/Code/Code.tsx

```diff
@@ -0,0 +1,96 @@
+import styled from 'styled-components';
+import { BaseText } from '../BaseText';
+import Box from '~components/Box';
+import type { TextTypes } from '~tokens/theme/theme';
+import { getPlatformType } from '~utils';
+
+export type CodeProps = {
+  /**
+   * children only supports string.
+   *
+   * ```tsx
+   * // ❌ Bad
+   * <Code><Box>something</Box></Code>
+   * ```
+   *
+   *```tsx
+   * // ✅ Good
+   * <Code>something</Code>
+   * ```
+   */
+  children: string;
+  /**
+   * Decides the fontSize and padding of Code
+   *
+   * @default medium
+   */
+  size?: 'large' | 'medium';
+  /**
+   * Decides the visibility and color of Code
+   *
+   * @default subtle
+   */
+  type?: TextTypes;
+};
+
+type CodeContainerProps = {
+  size: CodeProps['size'];
+  children: React.ReactNode;
```

Comment:
🤔 don't think you need to specify children for styled components.

Reply:
Oh right. Will remove it.

Reply:
done